impeller/renderer/backend/vulkan/gpu_tracer_vk.cc

```diff
@@ -121,9 +117,7 @@ void GPUTracerVK::RecordEndFrameTime() {
               "FrameTimeMS", gpu_ms);
         }
       })) {
-    if (!buffer->SubmitCommands()) {
-      VALIDATION_LOG << "GPUTracerVK failed to record frame end time.";
-    }
+    VALIDATION_LOG << "GPUTracerVK failed to record frame end time.";
```

Comment:
optional minor readability nit: Since the body of the if follows a long closure argument to `SubmitCommands` it was hard for me to tell whether it was part of the closure, part of the if-body, or after the if-body. Maybe consider assigning the result of `SubmitCommands` to a bool variable, and then branch on the bool variable.

Reply:
Done